components/drata/sources/vendor-updated/vendor-updated.mjs

```diff
@@ -7,7 +7,7 @@ const docsLink = "https://developers.drata.com/docs/openapi/reference/operation/
 export default {
   key: "drata-vendor-updated",
   name: "Vendor Updated",
-  description: `Emit new event when a vendor is updated. [See docs here](${docsLink}).`,
+  description: `Emit a new event when a vendor is updated. [See docs here](${docsLink}).`,
```

Comment:
```suggestion
  description: `Emit a new event when a vendor is updated. [See the documentation](${docsLink}).`,
```

Reply:
Changed "docs here" to "the documentation" to follow Google's Developer docs style guide.